Foods.yaml

```
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara"]
+  - ["Pizza", "Fermented whale blubber", "Squirrel", "Bratwurst", "kale_smoothies", "cheese", "pie", "Carbonara"]
```

it was pointed out to me yesterday that we should try to stick to `snake_case` in `yaml` files in the NUbots codebase
```suggestion
  - ["pizza", "fermented_whale_blubber", "squirrel", "bratwurst", "kale_smoothies", "cheese", "pie", "carbonara"]
```